rustls/src/webpki/mod.rs

```diff
@@ -3,12 +3,12 @@ use alloc::sync::Arc;
 use crate::error::{CertRevocationListError, CertificateError, Error};
 
 mod anchors;
-mod client_verifier_builder;
+mod verifier_builders;
```

Comment:
Alternative proposal: go the other way and split `client_verifier_builder` and `verify` into `client_verifier` and `server_verifier` builder modules that contain the builders? In general I don't love it when our builders live in a different file from the type they build.

Reply:
Sure, that seems sensible. I've reworked accordingly.